packages/opds-web-client/CHANGELOG.md

```diff
@@ -1,60 +1,85 @@
 ## Changelog
 
+### v0.3.1
+
+- added "prettier" code formatter and a git hook to run prettier before each commit
+- Updated tslint config to include prettier so that formatting is no longer a lint concern
+- ran prettier on all files, so many small file changes.
+
 ### v0.3.0
+
```

Comment:
Did prettier add these or did you?

Reply:
Prettier did it. We can disable prettier for markdown files if you like